packages/manager/modules/beta-preference/src/template.html

```diff
@@ -0,0 +1,12 @@
+<p data-translate="user_account_advanced_section_beta_description"></p>
+
+<div class="oui-box oui-box_light mb-5">
+    <oui-spinner data-ng-if="$ctrl.isUpdatingBeta" data-size="s"></oui-spinner>
+    <oui-checkbox
+        data-ng-if="!$ctrl.isUpdatingBeta"
+        aria-label=""
```

Comment:
Is it necessary ?

Reply:
Auto generated with @antleblanc vscode-ovh-ui-kit-snippets and forgot to remove it 😉

Reply:
Pay attention because all snippets are no more compatible with the ovh-ui-kit@2.0.0 (see: https://github.com/antleblanc/vscode-ovh-ui-kit-snippets/blob/master/CHANGELOG.md#200-2019-09-03).